packages/react-components/react-overflow/library/src/components/Overflow.tsx

```diff
@@ -20,6 +20,7 @@ export type OverflowProps = Partial<
   Pick<ObserveOptions, 'overflowAxis' | 'overflowDirection' | 'padding' | 'minimumVisible'>
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32909&commitId=aba8f117a9382fb96120ef328b04c2c724a58eaf&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|7| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1166| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|995| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2113| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>